drracket-tool-text-lib/drracket/private/syncheck/traversals.rkt

```diff
@@ -1144,39 +1144,39 @@
 ;; to do this, but maybe lets leave that for another day.
 (define (annotate-counts connections)
-  (for ([(key val) (in-hash connections)])
-    (when (list? val)
-      (define start (first val))
-      (define end (second val))
-      (define color? (third val))
-      (define (show-starts)
-        (when (zero? start)
-          (define defs-text (current-annotations))
-          (when defs-text
-            (send defs-text syncheck:unused-binder
-                  (list-ref key 0) (list-ref key 1) (list-ref key 2))))
-        (add-mouse-over/loc (list-ref key 0) (list-ref key 1) (list-ref key 2)
-                            (cond
-                              [(zero? start)
-                               (string-constant cs-zero-varrefs)]
-                              [(= 1 start)
-                               (string-constant cs-one-varref)]
-                              [else
-                               (format (string-constant cs-n-varrefs) start)])))
-      (define (show-ends)
-        (unless (= 1 end)
-          (add-mouse-over/loc (list-ref key 0) (list-ref key 1) (list-ref key 2)
-                              (format (string-constant cs-binder-count) end))))
-      (cond
-        [(zero? end)   ;; assume this is a binder, show uses
-         #;(when (and color? (zero? start))
-             (color-unused-binder (list-ref key 0) (list-ref key 1) (list-ref key 2)))
-         (show-starts)]
-        [(zero? start) ;; assume this is a use, show bindings (usually just one, so do nothing)
-         (show-ends)]
-        [else          ;; crazyness, show both
-         (show-starts)
-         (show-ends)]))))
+  (for ([(key val) (in-hash connections)]
+        #:when (list? val))
+    (define start (first val))
+    (define end (second val))
+    (define color? (third val))
+    (define (show-starts)
+      (when (zero? start)
+        (define defs-text (current-annotations))
+        (when defs-text
+          (send defs-text syncheck:unused-binder (list-ref key 0) (list-ref key 1) (list-ref key 2))))
+      (add-mouse-over/loc (list-ref key 0)
+                          (list-ref key 1)
+                          (list-ref key 2)
+                          (cond
+                            [(zero? start) (string-constant cs-zero-varrefs)]
+                            [(= 1 start) (string-constant cs-one-varref)]
+                            [else (format (string-constant cs-n-varrefs) start)])))
+    (define (show-ends)
+      (unless (= 1 end)
+        (add-mouse-over/loc (list-ref key 0)
+                            (list-ref key 1)
+                            (list-ref key 2)
+                            (format (string-constant cs-binder-count) end))))
+    (cond
+      ;; assume this is a binder, show uses
+      #;(when (and color? (zero? start))
+          (color-unused-binder (list-ref key 0) (list-ref key 1) (list-ref key 2)))
+      [(zero? end) (show-starts)]
```

Comment:
Something went wrong here. The code should behave the same but the comment went into the wrong place.

Reply:
This was intentional, but it's undesirable in this case. Will try to make an adjustment, thanks!